src/main/java/io/gravitee/policy/jwt/JWTPolicy.java

```diff
@@ -166,12 +166,14 @@ public void onRequest(Request request, Response response, ExecutionContext execu
     }
 
     private String getClientId(JWTClaimsSet claims) {
-        String clientId = null;
+        String clientId = (String) claims.getClaim(configuration.getClientIdClaim());
```

Comment:
What if the configured claim is an array?

Reply:
didn't know it was possible.
do you want a mechanism asit's done for audClaim ? 
https://github.com/gravitee-io/gravitee-gateway/pull/644/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R279-R292